### DISCUSSION TITLE
Can the color of the brewfest stein be changed?

### DISCUSSION THREAD

**Discussion by rugzoneruk** [chromiecraft/chromiecraft#6174] [Suggestions]
We had the green brewfest stein and it's feats of strength achievement last year. I was told that it stays the same since the client has the last official one on 3.3.5, which was green. Can it be changed so we can have a different colored stein this year/ later years?

**Comment by Amrx2** [comment]
Hi @rugzoneruk, whilst late to respond this change was implemented in game during Brewfest with new coloured steins being available this year. We will endeavour to remember to do the same for next year also!

Cheers,
Construct
